prime-dotnet-webapi/Configuration/Database/Seeded Tables/LicenseConfiguration.cs

```diff
@@ -109,6 +109,47 @@ public override IEnumerable<License> SeedData
                     new License { Code = 85, Weight = 4, Name = "Non-practicing registration" },
                     new License { Code = 86, Weight = 4, Name = "Temporary registration" },
 
+
+                    //College of Oral Health Professionals
+                    // Dental Assistant
+                    new License { Code = 93, Weight = 120, Name = "Full Certified Dental Assisant" },
+                    new License { Code = 94, Weight = 121, Name = "Limited Certified Dental Assistant" },
+                    new License { Code = 95, Weight = 123, Name = "Non-Practising Certified Dental Assistant" },
+                    new License { Code = 96, Weight = 124, Name = "Temporary Certified Dental Assistant" },
+
+                    // Dental Hygienist
+                    new License { Code = 97, Weight = 131, Name = "Registered Dental Hygienist" },
+                    new License { Code = 98, Weight = 132, Name = "Dental Hygiene Practitioner" },
+                    new License { Code = 99, Weight = 133, Name = "Non-Practising Dental Hygienist" },
+                    new License { Code = 100, Weight = 134, Name = "Temporary Dental Hygienist" },
+
+                    // Dental Technician
```

Comment:
Please check ordering of licence classes versus https://oralhealthbc.ca/register-renew/registrant-classifications/dental-technician/

Reply:
The last 3 licence classes have been re-ordered from Leanne in the excel spreadsheet. It is different that the order in web site.